SU2_CFD/include/solvers/CHeatSolver.hpp

```diff
@@ -38,12 +38,23 @@
  */
 class CHeatSolver final : public CSolver {
 protected:
+  static constexpr size_t MAXNDIM = 3; /*!< \brief Max number of space dimensions, used in some static arrays. */
+  static constexpr size_t MAXNVAR = 1; /*!< \brief Max number of variables, for static arrays. */
+
   unsigned short nVarFlow, nMarker, CurrentMesh;
-  su2double **HeatFlux, *HeatFlux_per_Marker, *Surface_HF, Total_HeatFlux, AllBound_HeatFlux,
-            *AverageT_per_Marker, Total_AverageT, AllBound_AverageT,
-            *Primitive, *Primitive_Flow_i, *Primitive_Flow_j,
-            *Surface_Areas, Total_HeatFlux_Areas, Total_HeatFlux_Areas_Monitor;
-  su2double ***ConjugateVar, ***InterfaceVar;
+  vector<vector<su2double> > HeatFlux;
```

Comment:
Maybe someone could add `/*!< \brief comments */`.

Reply:
Hmm yes I can do that, I already worked on the output of the heatsolver ... but I put that on the long bench. I am not sure if all those containers are necessary. 
I'll add a commit tomorrow 👍